### PR TITLE
tran_sock: use ERROR_INT()

### PR DESCRIPTION
**Pull request by jlevon**

### CODE REVIEW

File: lib/tran_sock.c [hunk 1]
```diff
@@ -391,61 +391,61 @@ tran_sock_init(vfu_ctx_t *vfu_ctx)
 {
     struct sockaddr_un addr = { .sun_family = AF_UNIX };
     tran_sock_t *ts = NULL;
-    int ret;
+    int err;
```
Comment (tmakatos) [inline lib/tran_sock.c:394]: Maybe it's best to keep `ret` since `err` is a function defined in `<err.h>`?

File: lib/tran_sock.c [hunk 2]
```diff
@@ -391,61 +391,61 @@ tran_sock_init(vfu_ctx_t *vfu_ctx)
...
-    ret = snprintf(addr.sun_path, sizeof(addr.sun_path), "%s", vfu_ctx->uuid);
-    if (ret >= (int)sizeof(addr.sun_path)) {
-        ret = -ENAMETOOLONG;
-    }
-    if (ret < 0) {
+    err = snprintf(addr.sun_path, sizeof(addr.sun_path), "%s", vfu_ctx->uuid);
+    if (err >= (int)sizeof(addr.sun_path)) {
+        err = ENAMETOOLONG;
+    } else if (err < 0) {
+        err = errno;
```
Comment (tmakatos) [inline lib/tran_sock.c:426]: According to the manpage `snprintf` doesn't set `errno`.

File: samples/client.c [hunk 3]
```diff
@@ -1155,8 +1136,7 @@ int main(int argc, char *argv[])
     ret = access_region(sock, VFU_PCI_DEV_CFG_REGION_IDX, false, 0, &config_space,
                         sizeof(config_space));
     if (ret < 0) {
-        errx(EXIT_FAILURE, "failed to read PCI configuration space: %s",
-             strerror(-ret));
+        errx(EXIT_FAILURE, "failed to read PCI configuration space");
```
Comment (tmakatos) [inline samples/client.c:1139]: `err` instead of `errx`?

**Review by tmakatos — APPROVED** [review]
Minor comments.

File: lib/tran_sock.c (hunk 1 above)
Comment (jlevon) [reply lib/tran_sock.c:394]: sure

File: lib/tran_sock.c (hunk 2 above)
Comment (jlevon) [reply lib/tran_sock.c:426]: good spot

File: samples/client.c (hunk 3 above)
Comment (jlevon) [reply samples/client.c:1139]: done

**Review by tmakatos — APPROVED** [review]

**Pull request merged**